db/migrate/20220527120826_add_index_on_category_slug_and_locale.rb

```diff
@@ -0,0 +1,6 @@
+class AddIndexOnCategorySlugAndLocale < ActiveRecord::Migration[6.1]
+  def change
+    add_column :categories, :slug, :string, null: false, default: 'common'
```

Comment:
What does the `default` mean here? This slug has to be unique for a (portal, locale), right?

Reply:
The slug values are not null, in that case, robocop gives an error that it should have some default values assigned to it, that's why added `common`, has no meaning as such.

Reply:
This means that if I added a category without a slug, it would update the slug as `common` right? This is not intended behavior. Can you please share the failing rubocop error?

Reply:
I removed the `common` setting it to `category` now or we could set it to the `blank` value.
Sorry, not RoboCop Postgres gives an error if we don't send any value, the following is the error.

`PG::NotNullViolation: ERROR:  column "slug" of relation "categories" contains null values`

Reply:
To resolve the rubocop error for the same https://github.com/rubocop/rubocop-rails/issues/237